### ISSUE TITLE
"No such file: gs://bucketname/input" even though directory exists

### ISSUE THREAD

**Issue by phiweger** [seqeralabs/nf-tower#283]
I upload a directory w/ two files "a" and "b" to some (google) bucket, eg:

```
gs://bucketname/input
- a.fq.gz
- b.fq.gz
```

I now run my wf (https://github.com/replikation/poreCov/tree/dev_replikation) via the Tower API, analogous to how I would specify it locally:

`nextflow run ... --fastq_raw gs://bucketname/input ...`

When I start this locally, the remote directory `gs://bucketname/input` is staged as expected and the wf starts.

When I run this via Tower, I get:

```
No such file: gs://bucketname/input
```

What am I doing wrong here?

I have all permissions in order, bc/ I can run another test wf on Tower just fine. However, this test run does not use an input directory but an input file like `gs://bucketname/some.fastq`.

Any help is greatly appreciated!

**Comment by phiweger** [comment]
Oddly, when I pass the input directory in using an `input.csv` with one row `name,gs://bucketname/input\n` and load this (specifically using the `nextflow ... --fastq_raw --list ...` equivalent over the API) then the wf starts.

**Comment by pditommaso** [comment]
Solved in the latest Tower release (21.02)

**Comment by phiweger** [comment]
thx!
